securedrop/tests/migrations/migration_fef03c1ec779.py

```diff
@@ -0,0 +1,40 @@
+from sqlalchemy import text
+from sqlalchemy.exc import OperationalError
+
+from db import db
+from journalist_app import create_app
+
+
+instance_config_sql = "SELECT * FROM instance_config"
+
+
+class UpgradeTester:
+    def __init__(self, config):
+        self.config = config
+        self.app = create_app(config)
+
+    def load_data(self):
+        pass
+
+    def check_upgrade(self):
+        with self.app.app_context():
+            db.engine.execute(text(instance_config_sql)).fetchall()
+
+
+class DowngradeTester:
+    def __init__(self, config):
+        self.config = config
+        self.app = create_app(config)
+
+    def load_data(self):
+        pass
+
+    def check_downgrade(self):
+        with self.app.app_context():
+            try:
+                db.engine.execute(text(instance_config_sql)).fetchall()
+
+            # The SQLite driver appears to return this rather than the
+            # expected NoSuchTableError.
+            except OperationalError:
```

Comment:
I also see this, NoSuchTableError is treated as an OperationalError:
```
    def do_execute(self, cursor, statement, parameters, context=None):
>       cursor.execute(statement, parameters)
E       sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such table: instance_config
E       [SQL: SELECT * FROM instance_config]
E       (Background on this error at: http://sqlalche.me/e/e3q8)

/opt/venvs/securedrop-app-code/lib/python3.5/site-packages/sqlalchemy/engine/default.py:552: OperationalError
```

Since the downgrade is a drop table action, I think this is fine.